operator/src/main/resources/scripts/wdt_create_filter.py

```diff
@@ -0,0 +1,23 @@
+def filter_model(model):
```

Comment:
- Missing copyright.  
- No code comments with a description.
- Consider renaming to 'model_create_filter.py' so it sorts next to the other model related scripts.

Reply:
@jshum2479 ?

Reply:
not renaming the script for now, but added comment and copyrights

Reply:
Please make the change or file a JIRA to track and link it here.

Reply:
I'll file a JIRA to track.  Resolving.